scripts/inventory/getinventory.py

```diff
@@ -22,23 +22,22 @@ def print_help(self, stream=sys.stderr):
         super(ArgParser, self).print_help(stream)
 
 
-def read_inventory_file(filename):
+def import_read_inventory_file():
     """
-    filename is a path to an ansible inventory file
+    This is a hack that makes this script dependent on commcare-hq-deploy
 
-    returns a mapping of group names ("webworker", "proxy", etc.)
-    to lists of hosts (ip addresses)
+    Not sure this is a great idea. If you ever find its brittleness breaks something,
+    feel free to copy and paste read_inventory_file from there, which is how it was before.
 
     """
-    from ansible.inventory import Inventory
-    from ansible.parsing.dataloader import DataLoader
-    from ansible.vars import VariableManager
-
-    inventory = Inventory(loader=DataLoader(), variable_manager=VariableManager(), host_list=filename)
-    return inventory.get_group_dict()
+    sys.path.append(os.path.join(ROOT, 'deployment', 'commcare-hq-deploy'))
+    from fab.utils import read_inventory_file
```

Comment:
Yeah, I agree this is better than copy-pasting. But I'd feel a little less awkward about this if we maybe caught ImportError and gave the poor dev who accidentally breaks this a hint about what they did wrong, maybe something like ...
```
try:
    from fab.utils import read_inventory_file
except ImportError:
    print('Ooops. It looks like the commcare-hq-deploy repo is no longer '
          'in "{}". Please update this script with its new location, or '
          'resolve the dependency.'.format(os.path.join(ROOT, 'deployment', 
          'commcare-hq-deploy')))
    raise
```